Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Handles/ShapeExtrusionHandle.cs

```diff
@@ -54,7 +54,7 @@ static Curve2D GetShape()
 
             var pointCount = s_ExtrusionMode ? s_Points.Count - 1 : s_Points.Count;
 
-            if ((s_Points[pointCount - 1] - s_Points[0]).sqrMagnitude < kDistanceEpsilon)
+            if ((s_Points[pointCount - 1] - s_Points[0]).sqrMagnitude < PointDrawing.kDistanceEpsilon)
```

Comment:
To ensure we have the same epsilon value in ShapeExtrusionHandle and PointDrawing (which shows the highlight) I've moved this to PointDrawing.